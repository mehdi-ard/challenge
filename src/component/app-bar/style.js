import style from 'provider/style';

const styles = (theme) => ({
  header: {
    // backgroundColor: '#fff !important',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
  },
  headerMenu: {
    backgroundColor: 'transparent',
  },
  menuHeader: {
    width: 'auto',
    borderLeft: 'none !important',
    borderRight: 'none !important',
  },
  headerMenuProfile: {
    display: 'flex',
    alignItems: 'flex-start',
    flexDirection: 'column',
    '& h5': {
      margin: 0,
      fontSize: 14,
      fontWeight: 'normal',
      lineHeight: 1,
      height: 20,
      textAlight: 'right',
    },
    '& span': {
      fontSize: 10,
      height: 20,
      display: 'flex',
      justifyContent: 'center',
      alignItems: 'center',
      textAlight: 'right',
    },
  },
});

export default style(styles);
