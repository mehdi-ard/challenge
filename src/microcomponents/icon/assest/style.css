@font-face {
  font-family: 'icomoon';
  src: url('fonts/icomoon.eot?xvmtyg');
  src: url('fonts/icomoon.eot?xvmtyg#iefix') format('embedded-opentype'),
    url('fonts/icomoon.ttf?xvmtyg') format('truetype'),
    url('fonts/icomoon.woff?xvmtyg') format('woff'),
    url('fonts/icomoon.svg?xvmtyg#icomoon') format('svg');
  font-weight: normal;
  font-style: normal;
  font-display: block;
}

[class^='icon-'],
[class*=' icon-'] {
  /* use !important to prevent issues with browser extensions that change fonts */
  font-family: 'icomoon' !important;
  speak: none;
  font-style: normal;
  font-weight: normal;
  font-variant: normal;
  text-transform: none;
  line-height: 1;

  /* Better Font Rendering =========== */
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
}

.icon-delete:before {
  content: '\e913';
}
.icon-down:before {
  content: '\e912';
}
.icon-archive:before {
  content: '\e900';
}
.icon-calendar:before {
  content: '\e901';
}
.icon-card:before {
  content: '\e902';
}
.icon-charity:before {
  content: '\e903';
}
.icon-clock:before {
  content: '\e904';
}
.icon-info:before {
  content: '\e905';
}
.icon-information:before {
  content: '\e906';
}
.icon-left:before {
  content: '\e907';
}
.icon-log-out:before {
  content: '\e908';
}
.icon-menu:before {
  content: '\e909';
}
.icon-minus:before {
  content: '\e90a';
}
.icon-pen:before {
  content: '\e90b';
}
.icon-phone:before {
  content: '\e90c';
}
.icon-plus:before {
  content: '\e90d';
}
.icon-right:before {
  content: '\e90e';
}
.icon-share:before {
  content: '\e90f';
}
.icon-user:before {
  content: '\e910';
}
.icon-wallet:before {
  content: '\e911';
}
