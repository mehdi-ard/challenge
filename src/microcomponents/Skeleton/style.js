import style from 'provider/style';

const styles = (theme) => ({
  innerSkeleton: {
    minHeight: 10,
    animationName: '$inner',
    animationDuration: '1.4s',
    animationTimingFunction: 'linear',
    animationIterationCount: 'infinite',
    background:
      '-webkit-gradient(linear, left top, right top, color-stop(25%, #f2f2f2), color-stop(37%, #e6e6e6), color-stop(63%, #f2f2f2))',
    background: 'linear-gradient(90deg, #f2f2f2 25%, #e6e6e6 37%, #f2f2f2 63%)',
    backgroundSize: '400% 100%',
  },

  '@keyframes inner': {
    '0%': { backgroundPosition: '100% 50%' },
    '100%': { backgroundPosition: '0 50%' },
  },
});

export default style(styles);
