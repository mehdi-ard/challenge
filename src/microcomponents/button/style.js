import style from 'provider/style';

const styles = (theme) => ({
  button: {
    // borderRadius: 5,
    // background: '#14cc8b',
    // height: 40,
    // color: theme.palette.text.contrastText,
    // border: 'none',
    '&:hover': {
      // background: '#14cc8b',
      // color: theme.palette.text.contrastText,
      // border: 'none',
    },
    '&:focus': {
      // background: '#14cc8b',
      // color: theme.palette.text.contrastText,
      // border: 'none',
    },
  },
});

export default style(styles);
