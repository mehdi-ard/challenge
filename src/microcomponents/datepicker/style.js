import style from 'provider/style';

const styles = (theme) => ({
  datepicker: {
    width: '100%',
    height: 48,
    color: theme.palette.text.primary,
    borderRadius: 5,
    // border: 'none',
  },
});

export default style(styles);
