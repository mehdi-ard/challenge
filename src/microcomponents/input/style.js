import style from 'provider/style';

const styles = (theme) => ({
  '@global': {
    '& .ant-input-affix-wrapper': {
      background: 'red',
    },
  },
  input: {
    '&:focus-within': {
      background: '#fff',
      border: '1px solid #c2c5c9',
      boxShadow: 'none !important',
    },
    '& input': {
      backgroundColor: '#fff',
      '&:focus': {
        background: '#fff',
      },
    },
    background: '#fff',
    borderRadius: 2,
    width: '100%',
    height: 48,
    display: 'flex',
    alignItems: 'center',
    '&:focus': {
      background: '#fff',
      border: '1px solid #c2c5c9',
      boxShadow: 'none !important',
    },
  },
});

export default style(styles);
