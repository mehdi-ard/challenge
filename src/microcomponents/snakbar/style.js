import style from 'provider/style';

const styles = (theme) => ({
  snackbar: {
    minWidth: 300,
    position: 'fixed',
    left: 10,
    bottom: 20,
    borderRadius: theme.shape.radius5,
  },
  alertClass: {
    background: 'transparent',
    border: 'none',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
    padding: '15px 20px !important',
    '& *': {
      position: 'relative',
      top: 0,
      left: 0,
      right: 0,
      bottom: 0,
      display: 'flex',
      alignItems: 'center',
      textAlign: 'right',
      color: '#fff',
    },
    '& .ant-alert-message': {
      width: '100%',
      paddingRight: 10,
    },
  },
  error: {
    background: '#f44336',
  },
  success: {
    background: '#4caf50',
  },
  warning: {
    background: '#ff9800',
  },
  info: {
    background: '#2196f3',
  },

  '@media screen and (max-width: 480px)': {
    snackbar: {
      width: '95%',
      left: 0,
      right: 0,
      bottom: 7,
      margin: 'auto',
      borderRadius: 0,
    },
  },
});

export default style(styles);
