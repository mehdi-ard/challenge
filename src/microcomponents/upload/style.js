import style from 'provider/style';

const styles = (theme) => ({
  root: {
    width: 450,
    margin: 'auto',
    border: `2px dashed ${theme.palette.primary.main}`,
    borderRadius: theme.shape.borderRadius,
    backgroundColor: theme.palette.grey[50],
    boxSizing: 'content-box',
    '& .filepond--panel-root': {
      backgroundColor: 'transparent',
    },
    height: 300,
    '& .filepond--drop-label': {
      height: '100% !important',
    },
    '& label': {
      height: '100%',
      width: '100%',
      display: 'flex !important',
      justifyContent: 'center',
      alignItems: 'center',
    },
  },
});

export default style(styles);
