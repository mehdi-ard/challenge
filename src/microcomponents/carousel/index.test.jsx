import React from 'react';
import renderer from 'react-test-renderer';
import Carousel from '.';
import './style.css';

class AppCarousel extends React.Component {
  state = {
    data: [
      {
        title: 'A',
        text:
          'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe cum illum commodi autem quo non aliquid reprehenderit, odit perferendis impedit explicabo nemo? Pariatur sed iusto exercitationem qui ab fuga atque.'
      },
      {
        title: 'B',
        text:
          'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe cum illum commodi autem quo non aliquid reprehenderit, odit perferendis impedit explicabo nemo? Pariatur sed iusto exercitationem qui ab fuga atque.'
      },
      {
        title: 'C',
        text:
          'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Saepe cum illum commodi autem quo non aliquid reprehenderit, odit perferendis impedit explicabo nemo? Pariatur sed iusto exercitationem qui ab fuga atque.'
      }
    ]
  };

  render() {
    const { data } = this.state;
    return (
      <>
        <Carousel data={data} {...this.props} />
      </>
    );
  }
}

it('renders default', () => {
  expect(renderer.create(<AppCarousel />).toJSON()).toMatchSnapshot();
});

it('renders autoplay', () => {
  expect(renderer.create(<AppCarousel autoplay />).toJSON()).toMatchSnapshot();
});

it('renders effect', () => {
  expect(
    renderer.create(<AppCarousel autoplay effect="fade" />).toJSON()
  ).toMatchSnapshot();
});

it('renders dotPosition', () => {
  expect(
    renderer.create(<AppCarousel dotPosition="left" />).toJSON()
  ).toMatchSnapshot();
});
