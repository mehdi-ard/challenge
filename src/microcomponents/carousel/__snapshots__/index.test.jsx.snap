// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders autoplay 1`] = `
<div
  className="ant-carousel"
>
  <div
    className="slick-slider slick-initialized"
    dir="ltr"
  >
    <div
      className="slick-list"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={null}
      onMouseMove={null}
      onMouseUp={[Function]}
      onTouchCancel={null}
      onTouchEnd={[Function]}
      onTouchMove={null}
      onTouchStart={[Function]}
      style={Object {}}
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "WebkitTransform": "translate3d(0px, 0px, 0px)",
            "WebkitTransition": "",
            "msTransform": "translateX(0px)",
            "opacity": 1,
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      >
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={-1}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={3}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={4}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={5}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
      </div>
    </div>
    <ul
      className="slick-dots slick-dots-bottom"
      onMouseEnter={null}
      onMouseLeave={null}
      onMouseOver={null}
      style={
        Object {
          "display": "block",
        }
      }
    >
      <li
        className="slick-active"
      >
        <button
          onClick={[Function]}
        >
          1
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          2
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          3
        </button>
      </li>
    </ul>
  </div>
</div>
`;

exports[`renders default 1`] = `
<div
  className="ant-carousel"
>
  <div
    className="slick-slider slick-initialized"
    dir="ltr"
  >
    <div
      className="slick-list"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={null}
      onMouseMove={null}
      onMouseUp={[Function]}
      onTouchCancel={null}
      onTouchEnd={[Function]}
      onTouchMove={null}
      onTouchStart={[Function]}
      style={Object {}}
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "WebkitTransform": "translate3d(0px, 0px, 0px)",
            "WebkitTransition": "",
            "msTransform": "translateX(0px)",
            "opacity": 1,
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      >
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={-1}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={3}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={4}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={5}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
      </div>
    </div>
    <ul
      className="slick-dots slick-dots-bottom"
      onMouseEnter={null}
      onMouseLeave={null}
      onMouseOver={null}
      style={
        Object {
          "display": "block",
        }
      }
    >
      <li
        className="slick-active"
      >
        <button
          onClick={[Function]}
        >
          1
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          2
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          3
        </button>
      </li>
    </ul>
  </div>
</div>
`;

exports[`renders dotPosition 1`] = `
<div
  className="ant-carousel ant-carousel-vertical"
>
  <div
    className="slick-slider slick-vertical slick-initialized"
    dir="ltr"
  >
    <div
      className="slick-list"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={null}
      onMouseMove={null}
      onMouseUp={[Function]}
      onTouchCancel={null}
      onTouchEnd={[Function]}
      onTouchMove={null}
      onTouchStart={[Function]}
      style={
        Object {
          "height": 0,
        }
      }
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "WebkitTransform": "translate3d(0px, 0px, 0px)",
            "WebkitTransition": "",
            "msTransform": "translateY(0px)",
            "opacity": 1,
            "transform": "translate3d(0px, 0px, 0px)",
            "transition": "",
          }
        }
      >
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={-1}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "outline": "none",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={3}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={4}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide slick-cloned"
          data-index={5}
          onClick={[Function]}
          style={
            Object {
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
      </div>
    </div>
    <ul
      className="slick-dots slick-dots-left"
      onMouseEnter={null}
      onMouseLeave={null}
      onMouseOver={null}
      style={
        Object {
          "display": "block",
        }
      }
    >
      <li
        className="slick-active"
      >
        <button
          onClick={[Function]}
        >
          1
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          2
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          3
        </button>
      </li>
    </ul>
  </div>
</div>
`;

exports[`renders effect 1`] = `
<div
  className="ant-carousel"
>
  <div
    className="slick-slider slick-initialized"
    dir="ltr"
  >
    <div
      className="slick-list"
      onClick={[Function]}
      onKeyDown={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={null}
      onMouseMove={null}
      onMouseUp={[Function]}
      onTouchCancel={null}
      onTouchEnd={[Function]}
      onTouchMove={null}
      onTouchStart={[Function]}
      style={Object {}}
    >
      <div
        className="slick-track"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        onMouseOver={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <div
          aria-hidden={false}
          className="slick-slide slick-active slick-current"
          data-index={0}
          onClick={[Function]}
          style={
            Object {
              "WebkitTransition": "opacity 500ms ease, visibility 500ms ease",
              "left": -0,
              "opacity": 1,
              "outline": "none",
              "position": "relative",
              "transition": "opacity 500ms ease, visibility 500ms ease",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={1}
          onClick={[Function]}
          style={
            Object {
              "WebkitTransition": "opacity 500ms ease, visibility 500ms ease",
              "left": -0,
              "opacity": 0,
              "outline": "none",
              "position": "relative",
              "transition": "opacity 500ms ease, visibility 500ms ease",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
        <div
          aria-hidden={true}
          className="slick-slide"
          data-index={2}
          onClick={[Function]}
          style={
            Object {
              "WebkitTransition": "opacity 500ms ease, visibility 500ms ease",
              "left": -0,
              "opacity": 0,
              "outline": "none",
              "position": "relative",
              "transition": "opacity 500ms ease, visibility 500ms ease",
              "width": 0,
            }
          }
          tabIndex="-1"
        >
          <div>
            <a
              style={
                Object {
                  "display": "inline-block",
                  "width": "100%",
                }
              }
              tabIndex={-1}
            >
              <img
                alt=""
              />
            </a>
          </div>
        </div>
      </div>
    </div>
    <ul
      className="slick-dots slick-dots-bottom"
      onMouseEnter={null}
      onMouseLeave={null}
      onMouseOver={null}
      style={
        Object {
          "display": "block",
        }
      }
    >
      <li
        className="slick-active"
      >
        <button
          onClick={[Function]}
        >
          1
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          2
        </button>
      </li>
      <li
        className=""
      >
        <button
          onClick={[Function]}
        >
          3
        </button>
      </li>
    </ul>
  </div>
</div>
`;
